views/lexicon/patterns/reporting/report.html.twig

```diff
@@ -248,136 +248,137 @@
             </table>
         </div>
     </div>
-    <div class="modal modal--large" id="schedule-add">
+    <div class="modal modal--large" id="schedule-add" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-description">
         <div class="modal__dialog">
           <div class="modal__content">
             <div class="modal__header">
-              <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
```

Comment:
Presuming this change will not cause hidden modals to be announced by screen readers?